pkg/gitserver/client.go

```diff
@@ -539,15 +539,29 @@ func (c *Client) IsRepoCloned(ctx context.Context, repo api.RepoName) (bool, err
 	return cloned, nil
 }
 
-// RepoInfo retrieves information about the repository on gitserver.
-//
-// The repository not existing is not an error; in that case, RepoInfoResponse.Cloned will be false
-// and the error will be nil.
+// RepoInfo is a helper for calling MultiRepoInfo with one repository. See that
+// method for details.
 func (c *Client) RepoInfo(ctx context.Context, repo api.RepoName) (*protocol.RepoInfoResponse, error) {
```

Comment:
To reduce unnecessary API surface expansion, I suggest we extend RepoInfo to take in a variable length slice of repo names.

```suggestion
func (c *Client) RepoInfo(ctx context.Context, repos ...api.RepoName) (*protocol.RepoInfoResponse, error) {
```